src/app/modules/group/http-services/get-requests.service.ts

```diff
@@ -27,13 +27,16 @@ export class GetRequestsService {
 
   getPendingRequests(
     groupId: string,
-    sort: string[] = []
+    sort: string[] = [],
+    includeSubgroup : boolean = false
   ): Observable<PendingRequest[]> {
     let params = new HttpParams();
     params = params.set('group_id', groupId);
     if (sort.length > 0) {
       params = params.set('sort', sort.join(','));
     }
+    if (includeSubgroup)
+      params = params.set('include_descendant_groups', '1');
```

Comment:
If no brackets, put it on a single line.
```suggestion
    if (includeSubgroup) params = params.set('include_descendant_groups', '1');
```